client/blocks/signup-form/style.scss

```diff
@@ -2,6 +2,53 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 
+body {
+	background: #fdfdfd;
+	.invite-logged-out-header {
```

Comment:
Can we consider moving this logic to something like `client/my-sites/invites/invite-accept/style.scss` ?
I think it should be the `invite-accept` job, as to not overcomplicate this component here.
But both ways work 👍

Reply:
It does make sense. 👍